engine/gamesys/src/gamesys/test/test_gamesys.h

```diff
@@ -610,8 +610,8 @@ void GamesysTest<T>::TearDown()
 
     dmGui::DeleteContext(m_GuiContext, m_ScriptContext);
     dmRender::DeleteRenderContext(m_RenderContext, m_ScriptContext);
-    dmGraphics::DeleteContext(m_GraphicsContext);
     dmJobThread::Destroy(m_JobThread);
+    dmGraphics::DeleteContext(m_GraphicsContext);
```

Comment:
The actual fix: Stop the thread first, before destroying the context.
Previously, the thread would continue to work with a deleted context, which isn't good.